fastlane/lanes/release.rb

```diff
@@ -555,17 +558,29 @@ def commit_version_and_build_files
   )
 end
 
-def create_release_management_pull_request(base_branch:, title:)
+def create_release_management_pull_request(release_version:, base_branch:, title:)
   token = ENV.fetch('GITHUB_TOKEN', nil)
 
   UI.user_error!('Please export a GitHub API token in the environment as GITHUB_TOKEN') if token.nil?
 
-  create_pull_request(
+  pr_url = create_pull_request(
     api_token: token,
     repo: 'wordpress-mobile/WordPress-iOS',
     title:,
     head: Fastlane::Helper::GitHelper.current_git_branch,
     base: base_branch,
     labels: 'Releases'
   )
+
+  # Next, set the milestone for the PR
+  #
+  # The create_pull_request action has a 'milestone' prameter, but it expects the milestone id.
+  # We don't know the id of the milestone, but we can use a different action to set it.
+  #
+  # PR URLs are in the format github.com/org/repo/pull/id
+  pr_number = pr_url.split('/').last
```

Comment:
I wonder if `File.basename` wouldn't make more sense here instead, semantically speaking?

```suggestion
  pr_number = File.basename(pr_url)
```

Reply:
I wish there was a `URI` `last_path_component` method. `File` seems out of place here.

Still, it's more elegant than manually chopping a string and picking the last element. 

Applying...